guides/common/attributes-satellite.adoc

```diff
@@ -136,27 +136,28 @@
 
 // For a release
 :RepoSatelliteVersion: {ProjectVersion}
+:project-client-2-name: satellite-6-client-2
 
 // RHEL 9 Satellite repos
 :RepoRHEL9ServerSatelliteCapsuleProjectVersion: satellite-capsule-{RepoSatelliteVersion}-for-rhel-9-x86_64-rpms
 :RepoRHEL9ServerSatelliteMaintenanceProjectVersion: satellite-maintenance-{RepoSatelliteVersion}-for-rhel-9-x86_64-rpms
 :RepoRHEL9ServerSatelliteServerProjectVersion: satellite-{RepoSatelliteVersion}-for-rhel-9-x86_64-rpms
-:RepoRHEL9ServerSatelliteToolsProjectVersion: satellite-client-6-for-rhel-9-<arch>-rpms
+:RepoRHEL9ServerSatelliteToolsProjectVersion: {project-client-2-name}-for-rhel-9-<arch>-rpms
 
 // RHEL 8 Satellite repos
 :RepoRHEL8ServerSatelliteCapsuleProjectVersion: satellite-capsule-{RepoSatelliteVersion}-for-rhel-8-x86_64-rpms
 :RepoRHEL8ServerSatelliteMaintenanceProjectVersion: satellite-maintenance-{RepoSatelliteVersion}-for-rhel-8-x86_64-rpms
 :RepoRHEL8ServerSatelliteMaintenanceProjectVersionPrevious: satellite-maintenance-{ProjectVersionPrevious}-for-rhel-8-x86_64-rpms
 :RepoRHEL8ServerSatelliteServerProjectVersion: satellite-{RepoSatelliteVersion}-for-rhel-8-x86_64-rpms
 :RepoRHEL8ServerSatelliteServerProjectVersionPrevious: satellite-{ProjectVersionPrevious}-for-rhel-8-x86_64-rpms
-:RepoRHEL8ServerSatelliteToolsProjectVersion: satellite-client-6-for-rhel-8-<arch>-rpms
+:RepoRHEL8ServerSatelliteToolsProjectVersion: {project-client-2-name}-for-rhel-8-<arch>-rpms
 :RepoRHEL8ServerSatelliteUtils: satellite-utils-{RepoSatelliteVersion}-for-rhel-8-x86_64-rpms
 
 // RHEL 7 Satellite repos
 :RepoRHEL7ServerSatelliteMaintenanceProjectVersion: rhel-7-server-satellite-maintenance-{RepoSatelliteVersion}-rpms
 :RepoRHEL7ServerSatelliteServerProjectVersion: rhel-7-server-satellite-{RepoSatelliteVersion}-rpms
 :RepoRHEL7ServerSatelliteServerProjectVersionPrevious: rhel-7-server-satellite-{ProjectVersionPrevious}-rpms
-:RepoRHEL7ServerSatelliteToolsProjectVersion: rhel-7-server-satellite-client-6-rpms
+:RepoRHEL7ServerSatelliteToolsProjectVersion: {project-client-2-name}-for-rhel-7-<arch>-rpms
```

Comment:
I would have expected `rhel-7-server-{project-client-2-name}-rpms` or something similar here.

Reply:
> I would have expected `rhel-7-server-{project-client-2-name}-rpms` or something similar here.

Agreed, @AkshayGadhaveRH can you fix it?

Reply:
Done